src/test/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipelineTest.java

```diff
@@ -71,9 +73,9 @@ public RuntimeConfigurablePipeline(PTransformSAM<?> tool, List<ReadFilter> filte
                 new Object[] { Lists.newArrayList(none), Collections.emptyList(), 0},
                 new Object[] { Collections.<ReadFilter>emptyList(), Lists.newArrayList(replaceBasesWithA), 7},
                 new Object[] { Lists.newArrayList(all), Lists.newArrayList(replaceBasesWithA), 7},
-                new Object[] { Lists.newArrayList(all), Lists.newArrayList(replaceBasesWithA.andThen(ReadTransformer.identity())), 7},
-                new Object[] { Lists.newArrayList(none.negate().and(all)), Lists.newArrayList(replaceBasesWithA.compose(ReadTransformer.identity())), 7},
-                new Object[] { Lists.newArrayList(none.negate(),all, all), Lists.newArrayList(replaceBasesWithA, replaceBasesWithA, ReadTransformer.identity()), 7}
+                new Object[] { Lists.newArrayList(all), Lists.newArrayList(replaceBasesWithA.andThen(UnaryOperator.identity())), 7},
```

Comment:
Is this change necessary/correct?  Intellij erroneously reports a compile error sometimes here, but I think we actually do want ReadTransformer.identity here.